bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```
@@ -185,6 +187,35 @@ class AudioManager {
     return this._outputDeviceId.value();
   }
 
+  set inputDevices(value) {
+    if (value && value.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)